app/models/notification.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Notification < ApplicationRecord
+  belongs_to :child
+  belongs_to :approval
+  # validates :approval_id, uniqueness: { scope: :child_id }
```

Comment:
How come you decided against this scoped uniqueness?

Reply:
I'm not sure if it is still necessary given that there should be only one notification per child since there is only one approval per child.